src/rate/rate.js

```diff
@@ -1,3 +1,45 @@
-export function formatRate(rate) {
-  return rate.toFixed(5);
+import config from '../config';
+import {
+  NUMBER_OF_EXCHANGE_RATE_SIGNIFICANT_DIGITS,
+  EXCHANGE_RATE_INVERSION_THRESHOLD,
+} from '../defaults';
+
+export function formatRate(exchangeRate, sourceCurrency, targetCurrency, options) {
+  const defaults = {
+    skipExchangeRateInversion: false,
+    numberOfSignificantDigits: NUMBER_OF_EXCHANGE_RATE_SIGNIFICANT_DIGITS,
+  };
+  const options = Object.assign({}, defaults, options);
```

Comment:
Could we use object spread here instead for readability?
```suggestion
  const options = { ...defaults, ...options };
```